neqo-common/src/codec.rs

```diff
@@ -15,6 +15,22 @@ pub struct Decoder<'a> {
 }
 
 impl<'a> Decoder<'a> {
+    /// Return the minimal encoded length of a varint with value `v`.
+    #[must_use]
+    pub const fn minimal_varint_len(v: u64) -> Option<usize> {
```

Comment:
I'd return a usize and panic if this was passed anything >= 2^62.  The only place you call this does the same.

`Encoder::varint_len()` already exists and does exactly that.

Reply:
Good suggestion. I'll also use the `match`-pattern there.